src/Take.Elephant.Sql/SqlHelper.cs

```diff
@@ -93,6 +94,20 @@ public static string GetLiteralJoinConditionStatement(IDatabaseDriver databaseDr
 
         }
 
+        public static string GetIsNotNullStatement(IDatabaseDriver databaseDriver, string[] columns)
+        {
+            return GetSeparateColumnsStatement(databaseDriver,
+                  databaseDriver.GetSqlStatementTemplate(SqlStatement.And),
+                  columns,
+                  columns, (d, c, p) => databaseDriver.GetSqlStatementTemplate(SqlStatement.QueryIsNotNull).Format(
```

Comment:
Since you are not using those lambda parameters, instead of using single letters, make explicit they are not important renaming to `_ `. 
And since there are 3 parameters, maybe write `(_, __, ___)`...